sway-lib-std/src/constants.sw

```diff
@@ -4,7 +4,35 @@ library;
 use ::contract_id::ContractId;
 
 /// The `BASE_ASSET_ID` represents the base asset of a chain.
-/// This is currently hard coded as a zero address, but will be configurable in the future.
+///
+/// # Additional Information
+///
+/// On the Fuel network, the base asset is Ethereum. It is hardcoded as the 0 ContractId.
```

Comment:
```suggestion
/// On the Fuel network, the base asset is Ether. It is hardcoded as the 0x00..00 ContractId.
```
Ether is the asset, Ethereum is the network. 

You can take my suggestion for 0x0 or leave it be.

Reply:
Or write `zero` instead of `0`